src/routes/docs/widget/roadmap/+page.md

```diff
@@ -1,37 +1,40 @@
-# Future APIs (not implemented)
+# Future Support (not yet implemented)
 
-## Currently **unsupported**
+## Planned for version 1.1
```

Comment:
I know we as a team have been using the term `1.1` to describe the "next" release after 1`.0. 

I think unless we decide to stop using semantic versioning, we should just describe future features, as just that. "Upcoming", "planned", etc as a new callback feature under semantic versioning would technically be a minor release.

I have tried to not use this terminology to break our planning and epics, but I do want us to be clear as under the current release strategy, 1.1 would almost certainly not contain all of these features.

Reply:
Fair enough. I've removed the specific numbering, but I do want to document that there are things for the near future and things that are not. If we don't hint towards this, I'm sure we'll have to field a lot of questions.